server/pbench/bin/index-pbench

```diff
@@ -321,6 +368,115 @@ class ResultData(object):
             datafiles.append(datafile)
         return datafiles
 
+    @staticmethod
+    def expand_template(templ, d):
```

Comment:
Is this going to fleshed out, or is it just a place holder for expanding the "uid" template field?

Reply:
Fleshed out a bit (but without error checking) in commit e8f89dc.